src/wallet/api/wallet2_api.h

```diff
@@ -1144,6 +1220,368 @@ struct Wallet
+    */
+    virtual std::uint32_t adjustPriority(std::uint32_t priority) = 0;
+    // QUESTION : Any suggestion for the function description below?
+    /**
+    * brief: coldTxAuxImport -
+    * param: ptx -
+    * param: tx_device_aux -
+    * note: sets status error on fail
+    */
+    virtual void coldTxAuxImport(const PendingTransaction &ptx, const std::vector<std::string> &tx_device_aux) const = 0;
+    /**
+    * brief: coldSignTx -
+    * param: ptx_in -
+    * outparam: exported_txs_out -
+    * param: dsts_info -
+    */
+    virtual void coldSignTx(const PendingTransaction &ptx_in, PendingTransaction &exported_txs_out, std::vector<cryptonote::address_parse_info> &dsts_info) const = 0;
+    /**
+    * brief: discardUnmixableOutputs - freeze all unmixable outputs
+    * note: sets status error on fail
+    */
+    virtual void discardUnmixableOutputs() = 0;
+    /**
+    * brief: setTxKey - set the transaction key (r) for a given <txid> in case the tx was made by some other device or 3rd party wallet
+    * param: txid -
+    * param: tx_key - secret transaction key r
+    * param: additional_tx_keys -
+    * param: single_destination_subaddress -
+    * note: sets status error on fail
+    */
+    virtual void setTxKey(const std::string &txid, const std::string &tx_key, const std::vector<std::string> &additional_tx_keys, const std::string &single_destination_subaddress) = 0;
+    /**
+    * brief: getAccountTags - get the list of registered account tags
+    * return: first.Key=(tag's name), first.Value=(tag's label), second[i]=(i-th account's tag)
+    */
+    virtual const std::pair<std::map<std::string, std::string>, std::vector<std::string>>& getAccountTags() const = 0;
+    /**
+    * brief: setAccountTag - set a tag to a set of subaddress accounts by index
+    * param: account_index - major index
+    * param: tag -
+    * note: sets status error on fail
+    */
+    virtual void setAccountTag(const std::set<uint32_t> &account_indices, const std::string &tag) = 0;
+    /**
+    * brief: setAccountTagDescription - set a description for a tag, tag must already exist
+    * param: tag -
+    * param: description -
+    * note: sets status error on fail
+    */
+    virtual void setAccountTagDescription(const std::string &tag, const std::string &description) = 0;
+    /**
+    * brief: exportOutputsToStr - export outputs and return encrypted data
+    * param: all   - go from `start` for `count` outputs if true, else go incremental from last exported output for `count` outputs (default: false)
+    * param: start - offset index in transfer storage, needs to be 0 for incremental mode (default: 0)
+    * param: count - try to export this amount of outputs (default: 0xffffffff)
+    * return: encrypted data as hex string if succeeded, else empty string
+    * note: sets status error on fail
+    */
+    virtual std::string exportOutputsToStr(bool all = false, std::uint32_t start = 0, std::uint32_t count = 0xffffffff) const = 0;
```

Comment:
Tricky naming conflict that we have here. We would like to switch from *output* to *enote* as term, and with that the method would get a name of `exportEnotesToStr`, yet the method `exportOutputs` exists, should probably continue to exist under that name, and sets a precedent of course.

Just now, in the mood that I am right now, I would probably choose `exportEnotesToStr`. If we are not a little bit aggressive in our switch to *enote* it will probably fail from pure inertia ...

Reply:
Anyway, I think if the name stays at least the description should use already *enote* as term.